.github/ISSUE_TEMPLATE/add-new-platform.md

```diff
@@ -0,0 +1,42 @@
+# Adding a new supported platform(N)
+
+## At Development
+
+- [ ] Create a WIP draft pull request on CoreOS-Assembler with new platform support [here](https://github.com/coreos/coreos-assembler)
```

Comment:
I think we can drop the redundant "WIP draft pull request" language?

"coreos-assembler", no cap.

Let's add high-level summaries of what each change needs to accomplish, and maybe a link to an example PR?

Reply:
I think I addressed this, let me know!